core/designsystem/src/main/kotlin/com/google/samples/apps/nowinandroid/core/designsystem/component/scrollbar/ScrollbarExt.kt

```diff
@@ -230,10 +230,5 @@ fun LazyStaggeredGridState.scrollbarState(
     return state
 }
 
-private inline fun <T> List<T>.floatSumOf(selector: (T) -> Float): Float {
-    var sum = 0f
-    for (element in this) {
-        sum += selector(element)
-    }
-    return sum
-}
+private inline fun <T> List<T>.floatSumOf(selector: (T) -> Float): Float =
+    fold(0f) { acc, it -> acc + selector(it) }
```

Comment:
Just trying to make this clearer for a first time reader
```suggestion
    fold(initial = 0f) { accumulator, listItem -> accumulator + selector(listItem) }
```